MREUnityRuntime/MREUnityRuntimeLib/Patching/Types/TransformPatch.cs

```diff
@@ -143,46 +143,104 @@ public virtual void RestoreAll()
 		}
 	}
 
-	public class ScaledTransformPatch : TransformPatch, IPatchable
+	/// <note>
+	/// Currently duplicates TransformPatch instead of inheriting so we can directly inherit from
+	/// Patchable to get those wins. Sadly C# does not support multiple inheritance or templated
+	/// mixins.
+	/// </note>
+	public class ScaledTransformPatch : Patchable<ScaledTransformPatch>, IPatchable
```

Comment:
If there's no shared code anymore, let's move `ScaledTransformPatch` to its own file.